src/stripeClient.ts

```diff
@@ -168,17 +161,11 @@ export class StripeClient {
   }
 }
 
-function getInstallPath(paths: string[]): string {
-  for (let i = 0; i < paths.length; i++) {
-    const path = paths[i];
-    try {
-      // eslint-disable-next-line no-sync
-      const isValidPath = fs.statSync(path).isFile();
-      if (isValidPath) {
-        return path;
-      }
-    } catch (err) {}
+function isFile(path: string): boolean {
+  try {
+    // eslint-disable-next-line no-sync
+    return fs.statSync(path).isFile();
```

Comment:
does this really need to be synchronous?

I'm not sure `.isFile` will do the right thing for you on symbolic links, you might need to `realpath` or `readlink` it, or you might just go ahead and `fs.openSync` and `fs.fstatSync` or something along those lines

Reply:
1. It does not need to be sync.
2. TIL `realPath` can resolve symbolic links 😮 

Will take a look at making these changes.